S/SoX/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SoX"
```

Comment:
What about `SoXR` or an even more clear `SoXResampler`?  I googled "sox" and couldn't find _anything_ related to this package in the first 8 pages of results

Reply:
Sure!

Reply:
Note that the name of the directory should be the same (and I'm actually surprised it didn't fail)

Reply:
Ok, thanks!